scripts/test-runner.js

```diff
@@ -388,12 +475,14 @@ class ArgsParseError {
 // Parses through the command line arguments and throws errors if usage is incorrect
 function argsParse(): ProgramArgs {
   let parsedArgs = minimist(process.argv.slice(2), {
-    string: ["filter"],
-    boolean: ["debugNames", "verbose"],
+    string: ["filter", "repl"],
```

Comment:
I do not feel "repl" is a very informative name regarding what it does. How about call it "outOfProcessRuntime"?